### ISSUE TITLE
Group and admin application not display in sessionbar in classic theme

### ISSUE THREAD

**Issue by ejouvin** [nuxsmin/sysPass#319]
When I am connected and use the classic theme, if the showPassIcon is set to false, the session bar does not display the user group name and the administration flag.

**Comment by nuxsmin** [comment]
Hi, I reviewing the current code in "sessionbar.inc" file and it should show the login info:

```
        <?php if ($showPassIcon): ?>
            <?php echo $userName, ' (', $userGroup, ') ', $adminApp; ?>
            <img src="imgs/key.png" class="iconMini"
                title="<?php echo _('Cambiar clave de usuario'); ?>"
                data-itemid="<?php echo $userId; ?>"
                Onclick="sysPassUtil.Common.usrUpdPass(this,<?php echo \SP\Controller\ActionsInterface::ACTION_USR_USERS_EDITPASS; ?>, '<?php echo $sk; ?>')" />
        <?php else: ?>
            <?php echo $userName, ' (', $userGroupName, ') ', $adminApp; ?>
<?php endif; ?>
```
I think that your pulled code is a code improvement, what do you think?

**Comment by ejouvin** [comment]
It is a little improvement but also a bug fixing. Take a look at both php echo.

In the first case
<?php echo $userName, ' (', $userGroup, ') ', $adminApp; ?>

In the second case
<?php echo $userName, ' (', $userGroup**Name**, ') ', $adminApp; ?>

And userGroupName seems to not exists.
In a first time, I replaced by the first one, but because I saw it is exactly the same in put in top.
And then i change to use the same code as the other theme
